build_projects/dotnet-host-build/PublishTargets.cs

```diff
@@ -280,6 +281,23 @@ public static BuildTargetResult PublishCoreHostPackages(BuildTargetContext c)
         }
 
         [Target]
+        public static BuildTargetResult PublishSignedPackages(BuildTargetContext c)
+        {
+            string signedPackages = EnvVars.EnsureVariable("SIGNED_PACKAGES");
+            if (signedPackages.Equals("true"))
+            {
+                foreach (var file in Directory.GetFiles(Dirs.Packages, "*.nupkg"))
```

Comment:
Yes, the existing host packages have their publish logic defined here - https://github.com/wtgodbe/core-setup/blob/c504f38fbb36f4f252c65a2f9bc7aa7e3c27bfb3/build_projects/dotnet-host-build/PublishTargets.cs#L271